.vscode-test.js

```diff
@@ -45,6 +48,16 @@ const extensions = [
 		label: 'github-authentication',
 		workspaceFolder: path.join(os.tmpdir(), `msft-auth-${Math.floor(Math.random() * 100000)}`),
 		mocha: { timeout: 60_000 }
+	},
+	{
+		label: 'vscode-api-tests',
+		workspaceFolder: path.join(__dirname, 'extensions', 'vscode-api-tests', 'testWorkspace'),
+		mocha: { timeout: 60_000 }
+	},
+	{
+		label: 'vscode-api-tests',
```

Comment:
I don't think the test runner right now handles multiple configurations with the same label; `yarn test-extension -l vscode-api-tests` is probably just running the first declaration of those tests. We should probably throw a more explicit error.

Reply:
In my experience it runs tests twice